exchanges/mock/recording.go

```diff
@@ -288,12 +288,20 @@ func CheckJSON(data interface{}, excluded *Exclusion) (interface{}, error) {
 	if reflect.TypeOf(data).String() == "[]interface {}" {
 		var sData []interface{}
 		for i := range data.([]interface{}) {
-			checkedData, err := CheckJSON(data.([]interface{})[i], excluded)
-			if err != nil {
-				return nil, err
-			}
+			v := data.([]interface{})[i]
+			switch v.(type) {
+			case map[string]interface{}, []interface{}:
+				checkedData, err := CheckJSON(v, excluded)
+				if err != nil {
+					return nil, err
+				}
 
-			sData = append(sData, checkedData)
+				sData = append(sData, checkedData)
+				continue
```

Comment:
I think we can remove this continue.

Reply:
Whoops! Ta, this was from an earlier revision of the change where the switch didn't have a default, and instead happened after. I'll remove it now